src/attr/_config.py

```diff
@@ -19,5 +22,8 @@ def set_run_validators(run):
 def get_run_validators():
     """
     Return whether or not validators are run.
+
+    .. deprecated:: 21.3.0 will not be moved to new ``attrs`` namespace.
+        Use :func:`attr.validators.get_disabled()` instead.
```

Comment:
```suggestion
    .. deprecated:: 21.3.0 It will not be removed, but it also will not be
        moved to new ``attrs`` namespace. Use `attr.validators.get_disabled()`
        instead.
```